### PR TITLE
I implemented restaurant list page

### PR DESCRIPTION
**Pull request by tvroom88**
Hi. I implemented restaurant list page. 

I made a adapter to order JsonObject data. 

Thank you

### CODE REVIEW

File: app/src/main/java/com/gogiyumandroid/MainActivity.kt [hunk 1]
```diff
@@ -26,4 +26,8 @@ class MainActivity : AppCompatActivity() {
     }
 }
 
-//https://gogiyum.com/food.json
\ No newline at end of file
+//https://gogiyum.com/food.json
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/MainActivity.kt]: Remove unnecessary comment.

File: app/src/main/java/com/gogiyumandroid/GoogleMap.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
+package com.gogiyumandroid
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/GoogleMap.kt:1]: You can keep this file now. But later don't add class files which is not used in this commit.

File: app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,78 @@
+package com.gogiyumandroid.RestaurantAdapter
+
+//class RestaurantAdapter(val _name: String, val _address: String, val _foods: Array<String> ) {
+class RestaurantAdapter(val _name: String, val _address: String, val _yrating: Double, val _grating: Double,
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt:3]: Can you explain how this class works?

File: app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt [hunk 4]
```diff
@@ -0,0 +1,78 @@
+package com.gogiyumandroid.RestaurantAdapter
+
+//class RestaurantAdapter(val _name: String, val _address: String, val _foods: Array<String> ) {
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt]: Remove unnecessary comment.

File: app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+import com.gogiyumandroid.RestaurantAdapter.RestaurantAdapter
+import kotlinx.android.synthetic.main.custom_restaurant_table_row.*
+import kotlinx.android.synthetic.main.custom_restaurant_table_row.view.*
+import kotlinx.android.synthetic.main.custom_table_row.view.*
+import okhttp3.*
+import org.json.JSONArray
+import java.net.URL
+
+
+@Suppress("DEPRECATION")
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt]: What is this Suppress for? Is this class for deprecation?

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt [hunk 6]
```diff
@@ -22,26 +24,28 @@ import java.io.IOException
 class HomeFragment : Fragment() {
 
     private lateinit var homeViewModel: HomeViewModel
-    private val myGogiyumAddress = "https://gogiyum.com/food.json"
+    private val myGogiyumAddress = "https://gogiyum.com/api/menu"
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt]: We usually use static const variable for fixed URL.

You can declare this variable outside of class.
```
const val MY_GOGIYUM_URL = "https://gogiyum.com/api/menu"

class HomeFragment : Fragment() {

}
```

File: app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt (hunk 3 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt:3]: After I get datas from url, I store datas as class form.

File: app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt (hunk 5 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt]: That part is not necessary anymore. so I erased it

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt (hunk 6 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt]: I changed it

File: app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt (hunk 4 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt]: okay. I erased unnecessary comment

File: app/src/main/java/com/gogiyumandroid/MainActivity.kt (hunk 1 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/MainActivity.kt]: I removed unnecessary comment

File: app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt (hunk 3 above)
Comment (jaehoonLee) [reply app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt:3]: Is this for tableview?

File: app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt (hunk 5 above)
Comment (jaehoonLee) [reply app/src/main/java/com/gogiyumandroid/RestaurantListActivity.kt]: Nice job.

**Comment by jaehoonLee** [comment]
Looks good to me.
Try to make CL smaller. 

실제 업무 때 CL이 길면, 애들이 코드 리뷰 잘 안해줌. 
그러면 프로젝트가 연기되고 더 느려져.

**Review by jaehoonLee — APPROVED** [review]

File: app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt (hunk 3 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt:3]: yes

**Pull request merged**